CanalTP/Chaos
# 2 new channels types additions

# Description

This PR adds 2 new channels, 'beacon' & 'type'

## Issue (optional)

Issue link: https://jira.canaltp.fr/browse/BOT-216

## Pull Request type (optional)

This is a bug fix / new feature?

## How to test

Here are the following steps to test this pull request:

- play migration `honcho run ./manage.py db upgrade`
- call a GET on channels API and check types
- then check you TrafficReport channels administration,
you should be able to use those new types
